rasa/core/policies/rule_policy.py

```diff
@@ -286,11 +286,8 @@ def _check_slots_fingerprint(
     ) -> Set[Text]:
```

Comment:
can we change the function name `_check_slots_fingerprint` to something like `_expected_but_missing_slots`?

Reply:
Yes!